Fabric.Authorization.API/scripts/Install-Authorization-Windows.ps1

```diff
@@ -248,19 +244,38 @@ function Get-ErrorFromResponse($response)
     return $responseBody
 }
 
-function Invoke-MonitorShallow($authorizationUrl)
-{
-	$url = "$authorizationUrl/_monitor/shallow"
-	Invoke-RestMethod -Method Get -Uri $url
+function Invoke-MonitorShallow($authorizationUrl) {
+    $url = "$authorizationUrl/_monitor/shallow"
+    Invoke-RestMethod -Method Get -Uri $url
 }
 
-if(!(Test-Path .\Fabric-Install-Utilities.psm1)){
-	Invoke-WebRequest -Uri https://raw.githubusercontent.com/HealthCatalyst/InstallScripts/master/common/Fabric-Install-Utilities.psm1 -OutFile Fabric-Install-Utilities.psm1
+function Get-EdwAdminUsersAndGroups() {
+    $results = Invoke-Sqlcmd -Query "
```

Comment:
Since this query will run on the web app machine, do we need to specify the ServerInstance to run against? I think this function also requires the sqlserver module, which has to be installed. We have a Invoke-Sql method in the Fabric-Install-Utilities.psm1 which we already import that doesn't have these pre-reqs. For a usage example see line 151 of this script.

Reply:
Yeah, you're right.

I tried using the Invoke-Sql method, but I was having a hard time getting results back (and I didn't see an example where that was happening). Maybe it's worth looking at again and possibly updating that...

Reply:
Let me know if you can't get the Invoke-Sql function to work and I can help take a look at it.

Reply:
Looking deeper, the Invoke-Sql function does an ExecuteNonQuery, so it won't return a result set. We can look into adding another function that does return a result set, I'm just concerned about introducing a dependency on the sqlserver powershell module.

Reply:
Yeah, I noticed that, too. Let me know what you think of this approach. I considered making a generic method in the Fabric-Install-Utilities module, but it can get pretty complex to pass in what columns you want returned from the query you pass in, and rather than over-engineering I decided to just put it inside the `Get-EdwAdminUsersAndGroups` function. It uses a connection string now, and not `Invoke-Sqlcmd`,.